### DISCUSSION TITLE
Server Error (500) - When deleting a project

### DISCUSSION THREAD

**Discussion by ktctomcat** [opengisch/QField#3852] [Q&A]
When I try to delete the project, "Server Error (500)" is displayed and I cannot delete it.
There is probably something wrong with the project file.
Anyone know how to deal with this?

**Comment by suricactus** [comment]
Deleting projects is temporary disabled for some accounts. Will enable them soon.

**Reply by ktctomcat** [reply]
I got it.
I'll try deleting it later.

thank you.

**Comment by martingeh** [comment]
Hi @suricactus, would you have any updates on when 'soon' might be?
Thanks.

**Reply by suricactus** [reply]
By the end of the week.
